kf_model_fhir/ingest_plugin/target_api_builders/group.py

```diff
@@ -0,0 +1,76 @@
+import pandas as pd
```

Comment:
Can you add module level docstrings to the target entity builders? Something as simple as: 

```python
"""
Builds a FHIR Group resource (https://www.hl7.org/fhir/group.html) from a row of data in table.
"""
```
and for the builders that build resources which conform to a KF profile:

```
"""
Builds a FHIR Patient resource (https://www.hl7.org/fhir/patient.html) from a row of data in a table. The resulting Patient resource will conform to the Kids First Patient profile. 
"""